main/main.c

```diff
@@ -830,16 +830,35 @@ PHPAPI ZEND_COLD void php_log_err_with_severity(const char *log_message, int sys
 			char *tmp;
 			size_t len;
 			zend_string *error_time_str;
+			time_t error_time = 0;
+			suseconds_t error_time_usec = 0;
+
+			const char *error_log_format = "d-M-Y H:i:s e";
+			size_t error_log_format_len = sizeof("d-M-Y H:i:s e") - 1;
+
+			if (PG(error_log_usec)) {
+				error_log_format = "d-M-Y H:i:s.u e";
+				error_log_format_len = sizeof("d-M-Y H:i:s.u e") - 1;
+			}
+
+#if HAVE_GETTIMEOFDAY
+			struct timeval tp = {0}; /* For setting microseconds */
+
+			gettimeofday(&tp, NULL);
+			error_time = tp.tv_sec;
+			error_time_usec = tp.tv_usec;
+#else
```

Comment:
It should be in the usec condition where this actually matters

Reply:
done, thanks. If you take a look once again, please?